packages/typescript/internal/BUILD.bazel

```diff
@@ -33,18 +33,13 @@ tsc(
     ],
 )
 
-bzl_library(
-    name = "bzl",
-    srcs = glob(
-        ["*.bzl"],
-        exclude = ["internal_*"],
-    ),
-    visibility = ["//:__subpackages__"],
-    deps = [
-        "@build_bazel_rules_nodejs//:bzl",
-        "@build_bazel_rules_nodejs//internal/common:bzl",
-        "@rules_nodejs//nodejs:bzl",
-    ],
+diff_test(
+    name = "check_validator.js",
+    # We are copying across packages here, so don't use generated_file_test, just print a command
```

Comment:
why not just move the .ts file to nodejs/private as well?

Reply:
because then we'd need rules in the BUILD file there to build it, and rules_nodejs has no way to compile typescript on its own